src/Navigation/Navigation.tsx

```diff
@@ -1,15 +1,21 @@
 import React from 'react'
 import './Navigation.scss'
-import {Link} from 'react-router-dom'
+import { Link } from 'react-router-dom'
 import heartIcon from '../logo-heart-icon.png'
 
-const Navigation = () => {
+interface NavProps {
+  darkMode: string;
+  toggleDarkMode: () => void;
+}
+
+const Navigation: React.FC<NavProps> = props => {
   return (
-    <nav className="navigation-bar">
-      <div className="behind-logo">
+    <>
+    <nav className={"navigation-bar" + props.darkMode}>
+      <div className={"behind-logo" + props.darkMode}>
         <Link className='title-link' to='/' >
           <h1 className="logo">
-            <div className='art-from-the'>
+            <div className={"art-from-the" + props.darkMode}>
```

Comment:
Passing down a part of a classname and then concatenating it an already created classname is such a creative way to toggle.